packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/util.ts

```diff
@@ -1,21 +1,20 @@
 import {SpeechRuleEngine} from "@khanacademy/mathjax-renderer";
-import * as SimpleMarkdown from "@khanacademy/pure-markdown";
-import {UnreachableCaseError} from "@khanacademy/wonder-stuff-core";
-
-import type {
-    LockedEllipseType,
-    LockedFigure,
-    LockedFigureColor,
-    LockedFigureFillType,
-    LockedFigureType,
-    LockedFunctionType,
-    LockedLabelType,
-    LockedLineType,
-    LockedPointType,
-    LockedPolygonType,
-    LockedVectorType,
-    LockedLineStyle,
+import {
+    type LockedEllipseType,
+    type LockedFigure,
+    type LockedFigureColor,
+    type LockedFigureFillType,
+    type LockedFigureType,
+    type LockedFunctionType,
+    type LockedLabelType,
+    type LockedLineType,
+    type LockedPointType,
+    type LockedPolygonType,
+    type LockedVectorType,
+    type LockedLineStyle,
+    mathOnlyParser,
```

Comment:
Why is the `mathOnlyParser` exported from `@khanacademy/perseus`? Could it live in `perseus-editor`?

Reply:
We need it in both! It's in Perseus for the visual TeX itself, and we need it in the editor for the aria label util.

Reply:
(see [other PR](https://github.com/Khan/perseus/pull/1875))